internal/calendar/google_calendar/calendar_test.go

```diff
@@ -1,159 +1,109 @@
 package googlecalendar
 
```

Comment:
Why are we going to commit a test file without code?  Wouldn't it be better to allocate a card to create this unit test?

Reply:
Sure, 
That´s the thing we´ve brought to discussion lately.
We decided to take out the entire test file, so It won´t cause an impact on our delivery and further deliveries.
The way we are coding our tests is creating an environment that will become hard to maintain
Also, We planned to create a card to keep it in mind. 
right @GLOFonseca ?

Reply:
We already have a card on our Staging board. We need to refactor all this package test, we have commented it with the idea of show that it`s important to prioritize this card.

I think it`s better than delete it, because if we delete it, maybe we will forget and keep without any test, like other Hellper packages.

Reply:
But the card is about this specific test or create a new coverage test in all files?

Reply:
It is about creating a new whole file that covers all methods.
The idea is to refactor all test file. This action probably will affect the main file (calender), that´s why we are skipping it.

Reply:
Hmmm it's okay with me